.travis.yml

```diff
@@ -33,6 +33,7 @@ script:
   # travis needs the config.py file ran owned by root in other environments it is the
   # securedrop_user default www-data
 - sudo chown root:root securedrop/config.py
+# replacing with my own stuff to try to get a log file written
```

Comment:
Not sure what this comment is explaining--perhaps an artifact from earlier troubleshooting?